hadoop-ozone/dist/src/main/compose/ozone/test.sh

```diff
@@ -50,6 +50,8 @@ execute_robot_test scm freon
 execute_robot_test scm cli
 execute_robot_test scm admincli
 
+execute_debug_tests
```

Comment:
@dombizita Looks like the CI fails because of after restart of datanode 2 , scm is still in safe mode and the remaining tests do not wait for safemode exit, We can either move this test to the end or add a wait_for_safemode call here.

Reply:
Hi @sadanand48, after some investigating I found what caused the failing. The admincli/pipeline.robot is creating a STANDALONE/ONE pipeline with one of the datanodes. After that in the debug tests I stop and reastart a datanode and I found that after that there won't be a RATIS/THREE pipeline, so the ozonfs tests are failing with the key uploading. It is happening because the pipeline limit is set to 1 on the datanodes and when we are trying to create a RATIS/THREE pipeline we are checking if the pipeline limit is reached on each datanode. When we are checking the current pipeline count on the datanodes we are deducting the single node pipelines, but we are only deducting the RATIS replication type ones. So we will count in the STANDALONE/ONE and with that we will reach the pipeline limit on that datanode, therefore we won't create the RATIS/THREE pipeline. I filed a JIRA (HDDS-6379) with a more detailed description.